Makefile

```diff
@@ -29,7 +30,7 @@ linux-builder:
 
 # The Linux target is built using a special Docker image, because this Makefile assumes the host
 # machine is running MacOS.
-dist/linux/amd64/$(PROJECT): $(EGOS) vendor
+dist/linux/amd64/$(PROJECT): init
```

Comment:
i think replacing `$(EGOS) vendor` with `init` here and below will defeat build caching for these steps.

maybe that's okay? the caching doesn't pay attention to changes in source files, so executing this step each time might be safest.

Reply:
I'm not sure which caching you are speaking about. It is really still doing the same thing I just made `$(EGOS)` run as a prerequisite of init target. the `glide.lock` caching still applies

Reply:
also to elaborate more I chose to use 'init' over 'vendor' since it now also runs the ego complies and I feel more truly initiates the project now

Reply:
The difference, as I understand it, is that previously the prerequisites for this rule were actual files on disk and `make` was able to check mtime to see if the prerequisites were satisfied. That's academic though, as I've talked myself out of the caching.

> 'init' over 'vendor' since it now also runs the ego complies and I feel more truly initiates the project now

💯